phpBB/styles/prosilver/theme/common.css

```diff
@@ -1276,13 +1325,15 @@ ul.linklist:after,
 	padding: 8px;
 }
 
-.dropdown-extended .footer > a {
+.dropdown-extended .footer > a,
+.smiley-content .footer  > a {
 	padding: 5px 0;
 }
 
 .dropdown-extended ul li a,
 .notification_list dt > a,
-.dropdown-extended .footer > a {
+.dropdown-extended .footer > a,
+.smiley-content .footer  > a {
 	text-decoration: none;
```

Comment:
no an updated screen shot of the posting editor and smiley drop-down you can edit the main pr message with them and just update them when ever you push changes